allennlp/models/constituency_parser.py

```diff
@@ -193,7 +216,6 @@ def forward(self,  # type: ignore
         # it for the validation and test sets.
         batch_gold_trees = [meta.get("gold_tree") for meta in metadata]
         if all(batch_gold_trees) and self._evalb_score is not None and not self.training:
-            # TODO(Mark): Predict POS and use here instead of using the gold ones.
```

Comment:
Why remove this TODO?

Reply:
Turns out that predicting pos tags only works with the elmo case, and that would be a bunch more work. Now the model is going to use embedded pos tags by default and the predictor will use Spacy's POS tagger (which is like 97.5% accurate anyway), so I thought it was not super important.